### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by lulu1694225**
Pls review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,5 +36,9 @@
                         .text(function(d) { return d; });
             });
         </script>
+        <table>
+            <tr> </tr>
```
Comment (HaoWang201) [inline index.html:40]: add some table data

**Review by HaoWang201 — CHANGES_REQUESTED** [review]
add some data

File: index.html (hunk 1 above)
Comment (lulu1694225) [reply index.html:40]: ok

**Review by HaoWang201 — APPROVED** [review]
asdf

**Pull request merged**